api/app/services/spree/webhooks/subscribers/queue_requests.rb

```diff
@@ -6,7 +6,7 @@ class QueueRequests
 
         def call(body:, event:)
           Spree::Webhooks::Subscriber.active.with_urls_for(event).each do |subscriber|
-            Spree::Webhooks::Subscribers::MakeRequestJob.perform_later(body, event, subscriber)
+            Spree::Webhooks::Subscribers::MakeRequestJob.perform_later(body.to_json, event, subscriber)
```

Comment:
@sebastian-palma I wonder, it's not recommended to push a lot of data as queue parameters, I wonder if this won't be a problem with bigger payloads, eg. if we would decide to by default include all associations for the record? To sum it up - are there any limits in the parameter size?

Reply:
I can't think of a limit size specifically @damianlegawiec, but we could modify this to avoid passing the body through the job arguments and instead to create the body in the job logic (HandleRequest) which is executed, so we could better follow the [best practices](https://github.com/mperham/sidekiq/wiki/Best-Practices#1-make-your-job-parameters-small-and-simple) for this.

Reply:
@sebastian-palma when we create the body in the job we're risking we're sending a newer state of that resource so probably sending the payload as it is, is better